.github/workflows/template_techdocs.yml

```diff
@@ -17,15 +17,15 @@ on:
         default: ${{ github.event.repository.name }}
     secrets:
       aws-access-key-id:
-        required: false
+        required: true
```

Comment:
We thought to keep this optional, so that a user can skip the publish step.
But maybe this is to implicit, we should introduce a flag like `skipPublish` instead.

Reply:
Is there a reason why you should skip the publish step? I have only seen the action and since we are not deploying to Azure, I thought that it makes sense to force the deployment to AWS. 🤔

Reply:
Yes, we want in PRs only build the Techdocs but not publish.